src/widget/emoticonswidget.cpp

```diff
@@ -165,29 +165,23 @@ void EmoticonsWidget::wheelEvent (QWheelEvent *e)
 {
     if (e->orientation() == Qt::Vertical)
     {
-     if (e->delta() < 0 )
-     {
-        stack.setCurrentIndex(stack.currentIndex()+1);
-     }
-     else
-     {
-            stack.setCurrentIndex(stack.currentIndex()-1);
-     }
-    emit PageButtonsUpdate();
+        if (e->delta() < 0)
+            stack.setCurrentIndex(stack.currentIndex() + 1);
+        else
+            stack.setCurrentIndex(stack.currentIndex() - 1);
 
+        emit pageButtonsUpdate();
     }
 }
 
-void EmoticonsWidget::PageButtonsUpdate()
+void EmoticonsWidget::pageButtonsUpdate() const
```

Comment:
>This method change children, not the object

If an object changes its children, the object is altered as well. As a rule of thumb, any method called something like add, udate, set, insert, etc. is not const. It doesn't matter, if that method incidentally doesn't change anything in its parent class.